### PR TITLE
Implemented News Feed using dummy data

### PR DESCRIPTION
**Pull request by BrandonMagana**
Created a simple screen to display News using Cards with an interactive button to display more/less information.

### CODE REVIEW

File: food-bank-app/src/components/NewsCard/NewsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import {
+  Image,
+  Text,
+  View,
+  StyleSheet,
+  TouchableOpacity
+} from 'react-native'
+import { News } from '../../models'
```
Comment (antonieto) [inline food-bank-app/src/components/NewsCard/NewsCard.tsx:8]: nit: wrong import order?

File: food-bank-app/src/components/NewsCard/NewsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        </Text>
+        <Text style={[style.date, { color: theme.colors.gray.date }]}>
+          { news.date.toLocaleDateString('es') }
+        </Text>
+        <Text
+          style={style.body}
+          numberOfLines={textShown ? undefined : 4}>
+            {news.body}
+        </Text>
+        <TouchableOpacity onPress={() => toggleShowText()}>
```
Comment (antonieto) [inline food-bank-app/src/components/NewsCard/NewsCard.tsx:44]:
```suggestion
        <TouchableOpacity onPress={toggleShowText}>
```

File: food-bank-app/src/screens/News/News.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    imageUrl: 'https://www.informador.mx/__export/1566991098442/sites/elinformador/img/2019/08/28/rs3243618_dig-2019-08-20-23-33-49_crop1566991097544.jpg_554688468.jpg'
+  },
+  {
+    title: 'El reto: acabar con el hambre',
+    date: new Date(),
+    body: testText
+  }
+]
+
+const Header = (): JSX.Element => {
```
Comment (antonieto) [inline food-bank-app/src/screens/News/News.tsx:33]: Is another component necessary?

**Review by antonieto — APPROVED** [review]
Looks good. Only suggested a few style comments.

File: food-bank-app/src/screens/News/News.tsx (hunk 3 above)
Comment (BrandonMagana) [reply food-bank-app/src/screens/News/News.tsx:33]: Not really, it is just for readability, i could change it though. What you think?

Comment (antonieto) [reply food-bank-app/src/screens/News/News.tsx:34]: Again, this is pure style, but [AirBnB convention](https://github.com/airbnb/javascript/blob/master/react/README.md) says one component per file:

> Only include one React component per file.

It's up to you, I don't think this is a reason to slow down development

Comment (BrandonMagana) [reply food-bank-app/src/screens/News/News.tsx]: I corrected this

**Pull request merged**